lib/flutter_html.dart

```diff
@@ -81,8 +82,51 @@ class Html extends StatelessWidget {
     this.style = const {},
     this.navigationDelegateForIframe,
   }) : data = null,
+        _selectable = false,
         assert(document != null),
-  anchorKey = GlobalKey(),
+        anchorKey = GlobalKey(),
+        super(key: key);
+
+  Html.selectable({
```

Comment:
I think you should add here the description and restrictions of this selectable widget. The description is in the readme, but the public interfaces should also be documented.